### DISCUSSION TITLE
The "Edit Pragmas" Tab is nearly not translateable

### DISCUSSION THREAD

**Discussion by Stheel83** [sqlitebrowser/sqlitebrowser#3531] [General]
Hi
nearly the whole "Edit Pragmas" Tab is not translated, not in any language I tested and it looks like it is not translatable.
(Only the popup, when you hover over the check box of "Case Sensitive Like" is translatable)
![Pragmas](https://github.com/sqlitebrowser/sqlitebrowser/assets/153910911/21fbbf13-cb79-4e2b-b5e6-dd7f7b1537e2)

I found hints to the Pragma-Tab in
MainWindow.cpp, MainWindow.h and

MainWindow.ui (please see Line 275)
`<string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Checkpoint Full FSYNC &lt;a href=&quot;https://www.sqlite.org/pragma.html#pragma_checkpoint_fullfsync&quot;&gt;&lt;img src=&quot;:/icons/whatis&quot;/&gt;&lt;/a&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>`

I am not a programmer, but that looks not so easy to change.

May it be possible to change something, so that it is translatable?

**Comment by mgrojo** [comment]
If I don't recall wrongly, I think we made that on purpose. These are the names of the pragmas in the technical documentation of SQLite and we wanted to keep them as they are for recognition. Nevertheless, most of them are difficult to translate. The same should apply to "Case Sensitive Like", that inconsistency is probably an overlook.

**Reply by Stheel83** [reply]
Ok, thanks for the reply and the clarification.
So you mean the popup should be reverted to english?

**Reply by mgrojo** [reply]
I was referring to the pragma names. If you mean this tooltip:
> Warning: this pragma is not readable and this value has been inferred. Writing the pragma might overwrite a redefined LIKE provided by an SQLite extension.

That is a warning to the user and is expected to be translated. I thought you meant the string "Case Sensitive Like" itself, which is a [pragma name](https://www.sqlite.org/pragma.html#pragma_case_sensitive_like). By the way, this pragma is now deprecated and SQLite says:
>  Older applications should discontinue use of this pragma at the earliest opportunity.

So we should do that after the next release, but there's no mention of an alternative.

**Reply by Stheel83** [reply]
Ok, no i only asked why the tooltip was translated. 
In German are all pragma options are in English, as it is intended.

So i will not do anything here and will see what you and the other programmer do with this pragma option in the future.

Thank you too for helping me.

**Comment by mgrojo** [comment]
By the way, pragma names and values are translatable now. See conversation in #3697

**Reply by Stheel83** [reply]
Ok thanks! i will take a look!
